seqr/views/apis/report_api.py

```diff
@@ -31,6 +31,10 @@
 MONDO_BASE_URL = 'https://monarchinitiative.org/v3/api/entity'
 
 
+airtable_enabled_analyst_required = active_user_has_policies_and_passes_test(
```

Comment:
should this live in `permissions_utils.py`?

Reply:
For permissions that are specific to a single file we tend to put them directly in the file, like for `anvil_workspace_access_required` which is only in the anvil_workspace_api file. So it could go either way